README.md

```diff
@@ -31,7 +44,7 @@ To validate your downloaded file's integrity, compare the lines in checksum.txt
 Clone the repo, and use a simple `npm install`.  You can then use a `npm run
 build` to create an optimised static build (serve it with e.g. [serve][serv]).
 
-### General notes
+### General Notes
```

Comment:
*grumbles about capitalists*

imo capitalizing Every Single Word in a header is redundant with the fact that it's a header. They're not proper names, or titles, and we aren't writing German, so headers should only have a capital at the start.  
Same for the other two headers you changed.

If you care a lot, I'll give you this. It's not my hill to die on. But if you're lukewarm on the issue I'd prefer reverting to less capitals.

Reply:
I almost added "fix Euro mixed caps O_0" to the commit message. This is an *American* company. Sorry.

In seriousness, I see we don't use all caps in other READMEs, so I'll accept the revert for consistency. I may die on this hill, however.